### PR TITLE
feat: Poc requirements

### PR DESCRIPTION
**Pull request by fabrizioacn**

### CODE REVIEW

File: src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/dao/PNEntity.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.swclient.mil.paymentnotice.dao;
+
+import org.bson.codecs.pojo.annotations.BsonId;
+
+import io.quarkus.mongodb.panache.common.MongoEntity;
+import it.gov.pagopa.swclient.mil.paymentnotice.bean.PspInfo;
+
+@MongoEntity(database = "mil", collection = "paymentnotice")
```
Comment (antoniotarricone) [inline src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/dao/PNEntity.java:8]: Please, use meaningful name: this is a PSP configuration!
Rename the entity to PspConfEntity and the Mongo collection to pspconf.

File: src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/dao/PNRepository.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.swclient.mil.paymentnotice.dao;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoRepository;
+
+@ApplicationScoped
+public class PNRepository implements ReactivePanacheMongoRepository<PNEntity> {
```
Comment (antoniotarricone) [inline src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/dao/PNRepository.java:8]: Use meaningful names!
Rename to PspConfRepository.

File: src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+public class ClosePaymentRequest {
+
+	private List<String> paymentTokens;
+	private String outcome;
+	private String idPsp;
+	private String pspBroker;
+	private String idChannel;
+	private String paymentMethod;
+	private String transactionId;
+	private String totalAmount;
```
Comment (antoniotarricone) [inline src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java:14]: This is an amount, why is it a string?

File: src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	private String fee;
```
Comment (antoniotarricone) [inline src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java:15]: This is an amount, why is it a string?

File: src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+import java.util.List;
+
+public class ClosePaymentRequest {
+
+	private List<String> paymentTokens;
+	private String outcome;
+	private String idPsp;
+	private String pspBroker;
+	private String idChannel;
+	private String paymentMethod;
```
Comment (antoniotarricone) [inline src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java:12]: This is an enum!

File: src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java [hunk 6]
```diff
@@ -0,0 +1,93 @@
+package it.gov.pagopa.swclient.mil.paymentnotice.bean;
+
+import java.util.List;
+
```
Comment (antoniotarricone) [inline src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/bean/ClosePaymentRequest.java:4]: In general: validation is missing!

**Review by antoniotarricone — COMMENTED** [review]
This is a partial review. According to given feedbacks, review the whole project. Pay attention to code formatting and to comments that are almost missing.

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,295 @@
...
+		<maven.compiler.release>${java.version}</maven.compiler.release>
+		<maven.compiler.target>${java.version}</maven.compiler.target>
+		<maven.compiler.source>${java.version}</maven.compiler.source>
+		<compiler-plugin.version>3.8.1</compiler-plugin.version>
+		<surefire-plugin.version>3.0.0-M7</surefire-plugin.version>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+		<quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+		<quarkus.platform.version>2.14.3.Final</quarkus.platform.version>
```
[the remark this answers is not in the document]
Comment (dsabacn) [reply pom.xml]: Updated

**Pull request merged**